transformers/component_tests/cat/configured_transformer_test.go

```diff
@@ -39,17 +38,16 @@ import (
 var _ = Describe("Executing the transformer", func() {
 	var (
 		db              = test_config.NewTestDB(test_config.NewTestNode())
-		contractAddress = test_data.CatAddress()
-		keccakOfAddress = types.HexToKeccak256Hash(contractAddress)
+		contractAddress = common.HexToAddress(test_data.CatAddress())
 		transformer     storage.Transformer
 		header          = fakes.FakeHeader
 	)
 
 	BeforeEach(func() {
-		storageKeysLookup := storage.NewKeysLookup(cat.NewKeysLoader(&mcdStorage.MakerStorageRepository{}, contractAddress))
-		repository := cat.StorageRepository{ContractAddress: contractAddress}
+		storageKeysLookup := storage.NewKeysLookup(cat.NewKeysLoader(&mcdStorage.MakerStorageRepository{}, contractAddress.Hex()))
+		repository := cat.StorageRepository{ContractAddress: contractAddress.Hex()}
 		transformer = storage.Transformer{
-			Address:           common.HexToAddress(contractAddress),
+			Address:           common.HexToAddress(contractAddress.Hex()),
```

Comment:
What type is `contractAddress`? I'm not sure I follow all the converting here. Isn't it already an `address`?

Reply:
yes, nice catch!